### ISSUE TITLE
Endless DB upgrade loop when upgrading.

### ISSUE THREAD

**Issue by 80kk** [nilsteampassnet/TeamPass#2457]
### Steps to reproduce
1. Clone the latest TeamPass 2.1.27.26
2. Run upgrade against 2.1.27.19
3. Get stuck in endless DB upgrade loop.

### Expected behaviour
Upgrade DB.

### Actual behaviour
Got stuck in endless DB upgrade loop: https://ibb.co/mcq1k0

### Server configuration
**Operating system**: CentOS 7.5.1804

**Web server:** Apache 2.4.6

**Database:** 5.5.60-MariaDB

**PHP version:** 7.0.30

**Teampass version:** 2.1.27.19

**Teampass configuration file:** 

**Updated from an older Teampass or fresh install:** Updated
```
<?php
global $SETTINGS;
$SETTINGS = array (
    'max_latest_items' => '10',
    'enable_favourites' => '1',
    'show_last_items' => '1',
    'enable_pf_feature' => '1',
    'log_connections' => '0',
    'log_accessed' => '1',
    'time_format' => 'H:i:s',
    'date_format' => 'd/m/Y',
    'duplicate_folder' => '1',
    'item_duplicate_in_same_folder' => '1',
    'duplicate_item' => '1',
    'number_of_used_pw' => '3',
    'manager_edit' => '1',
    'cpassman_dir' => '/var/www/html',
    'cpassman_url' => 'https://teampass.domain.com',
    'favicon' => 'https://teampass.domain.com/favicon.ico',
    'path_to_upload_folder' => '/var/www/html/upload',
    'url_to_upload_folder' => 'https://teampass.domain.com/upload',
    'path_to_files_folder' => '/var/www/html/files',
    'url_to_files_folder' => 'https://teampass.domain.com/files',
    'activate_expiration' => '1',
    'pw_life_duration' => '0',
    'maintenance_mode' => '0',
    'enable_sts' => '0',
    'encryptClientServer' => '1',
    'cpassman_version' => '2.1.27',
    'ldap_mode' => '0',
    'ldap_type' => '0',
    'ldap_suffix' => '0',
    'ldap_domain_dn' => '0',
    'ldap_domain_controler' => '0',
    'ldap_user_attribute' => '0',
    'ldap_ssl' => '0',
    'ldap_tls' => '0',
    'ldap_elusers' => '0',
    'ldap_search_base' => '0',
    'ldap_port' => '389',
    'richtext' => '0',
    'allow_print' => '0',
    'roles_allowed_to_print' => '0',
    'show_description' => '1',
    'anyone_can_modify' => '1',
    'anyone_can_modify_bydefault' => '0',
    'nb_bad_authentication' => '0',
    'utf8_enabled' => '1',
    'restricted_to' => '1',
    'restricted_to_roles' => '1',
    'enable_send_email_on_user_login' => '0',
    'enable_user_can_create_folders' => '1',
    'insert_manual_entry_item_history' => '0',
    'enable_kb' => '1',
    'enable_email_notification_on_item_shown' => '0',
    'enable_email_notification_on_user_pw_change' => '0',
    'custom_logo' => 'https://www.domain.com/logo.png',
    'custom_login_text' => '',
    'default_language' => 'english',
    'send_stats' => '0',
    'send_statistics_items' => 'stat_country;stat_users;stat_items;stat_items_shared;stat_folders;stat_folders_shared;stat_admins;stat_managers;stat_ro;stat_mysqlversion;stat_phpversion;stat_teampassversion;stat_languages;stat_kb;stat_suggestion;stat_customfields;stat_api;stat_2fa;stat_agses;stat_duo;stat_ldap;stat_syslog;stat_stricthttps;stat_fav;stat_pf;',
    'send_stats_time' => '1523461209',
    'get_tp_info' => '1',
    'send_mail_on_user_login' => '0',
    'nb_items_by_query' => 'auto',
    'enable_delete_after_consultation' => '1',
    'enable_personal_saltkey_cookie' => '0',
    'personal_saltkey_cookie_duration' => '31',
    'email_smtp_server' => '',
    'email_smtp_auth' => '',
    'email_auth_username' => '',
    'email_auth_pwd' => '',
    'email_port' => '',
    'email_security' => '',
    'email_server_url' => '',
    'email_from' => '',
    'email_from_name' => '',
    'pwd_maximum_length' => '120',
    'google_authentication' => '0',
    'delay_item_edition' => '0',
    'allow_import' => '1',
    'proxy_ip' => '',
    'proxy_port' => '',
    'upload_maxfilesize' => '50mb',
    'upload_docext' => 'doc,docx,dotx,xls,xlsx,xltx,rtf,csv,txt,pdf,ppt,pptx,pot,dotx,xltx',
    'upload_imagesext' => 'jpg,jpeg,gif,png',
    'upload_pkgext' => '7z,rar,tar,zip,gzip,bz2,gz',
    'upload_otherext' => 'sql,xml,sh,conf,cnf,yml,yaml,json,tpm,textClipping,cer,pem,key,pub',
    'upload_imageresize_options' => '1',
    'upload_imageresize_width' => '1024',
    'upload_imageresize_height' => '768',
    'upload_imageresize_quality' => '90',
    'use_md5_password_as_salt' => '0',
    'ga_website_name' => 'TeamPass for ChangeMe',
    'api' => '0',
    'subfolder_rights_as_parent' => '1',
    'show_only_accessible_folders' => '0',
    'enable_suggestion' => '0',
    'otv_expiration_period' => '1',
    'default_session_expiration_time' => '60',
    'duo' => '0',
    'enable_server_password_change' => '0',
    'ldap_object_class' => '0',
    'bck_script_path' => '/var/www/html/backups',
    'bck_script_filename' => 'bck_teampass',
    'syslog_enable' => '0',
    'syslog_host' => 'localhost',
    'syslog_port' => '514',
    'manager_move_item' => '0',
    'create_item_without_password' => '1',
    'otv_is_enabled' => '1',
    'agses_authentication_enabled' => '0',
    'item_extra_fields' => '0',
    'saltkey_ante_2127' => 'none',
    'migration_to_2127' => 'done',
    'files_with_defuse' => 'done',
    'timezone' => 'Europe/London',
    'enable_attachment_encryption' => '1',
    'personal_saltkey_security_level' => '50',
    'ldap_new_user_is_administrated_by' => '0',
    'disable_show_forgot_pwd_link' => '0',
    'offline_key_level' => '60',
    'enable_http_request_login' => '0',
    'ldap_and_local_authentication' => '0',
    'secure_display_image' => '1',
    'can_create_root_folder' => '1',
    'copy_to_clipboard_small_icons' => '1',
    'tree_counters' => '0',
    'teampass_version' => '2.1.27',
    'upload_zero_byte_file' => '1',
    'upload_all_extensions_file' => '1',
    );
```

### Client configuration
**Browser:** FF 61.0.1

**Operating system:** Ubuntu 17.10

#### Web server error log
```
192.170.2.126 - - [31/Oct/2018:14:54:12 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:12 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:13 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:13 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:13 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:13 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:14 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:14 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:14 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:14 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:15 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:15 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:15 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:16 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:16 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:16 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:16 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:17 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:17 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:17 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:18 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:18 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:18 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:18 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:19 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:19 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:19 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:20 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:20 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:20 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:20 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:21 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:21 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:21 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:22 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.1.222 - - [31/Oct/2018:14:54:22 +0000] "GET /api/health HTTP/1.1" 404 208 "-" "ELB-HealthChecker/2.0"
192.170.2.126 - - [31/Oct/2018:14:54:22 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:22 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:22 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:23 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:23 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:23 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:24 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:24 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:24 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:24 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:25 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:25 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:25 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:26 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:26 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:26 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:27 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:27 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:27 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:27 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:28 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:28 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:28 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:29 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:29 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"
192.170.2.126 - - [31/Oct/2018:14:54:29 +0000] "POST /install/upgrade_run_2.1.27.php HTTP/1.1" 200 185 "https://teampass.domain.com/install/upgrade.php" "Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0"

```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by nilsteampassnet** [comment]
Can you please share the errors from the apache log?
If there is an endless loop then the error is captured in the logs.

The list provided just provides the accesses.

**Comment by nilsteampassnet** [comment]
OK

How long did you wait?
How many items do you manage in the database?

**Comment by 80kk** [comment]
I was was waiting about 20 minutes. I have about 300 items. That never took so long, usually about 3-5 seconds on previous updates.

**Comment by prozit** [comment]
I had the same issue when I upgraded from 2.1.26 to 2.1.27.x
In my case the issue was caused because I already ran one update before on the same date.
there were two files that were being renamed in the teampass web folder (tp.config.php and csfrp.config.php if I am not mistaking). Those files were renamed to a file which received an extention of the current date. Because those files were already renamed to the backup files during the first upgrade cycle, the couldn't rename anymore during the second upgrade cycle because the backup files already exited.

An option could be to rename including the time of the upgrade or by using unix timestamp, so each upgrade receives a unique renaming file extension.

It is possible you are experiencing another issue, however, I was only able to figure this out by using the firefox developer tools (Web Developer, Network). Once the script starts looping, in the response tab, I saw an error indicating the issue with the already existing file...

This didn't explicitly indicate a visual error in the web browser while it did using the web developer tools, so this could help you search for the specific issue you are encountering...

Good luck with your search!
Best regards,
Francis Claessens.
